R/mergeReport.r

```diff
@@ -168,6 +168,24 @@ print.egret <- function(x,...){
 
 }
 
+#' Plots EGRET object
+#' 
+#' Plot function for EGRET object (calls multiPlotDataOverView).
+#' 
+#' @param eList named list with at least Daily, Sample, and INFO dataframes
+#' @param \dots any other arguments to be passed to \code{multiPlotDataOverview}
+#' 
+#' @method plot egret
+#' @export
+#' 
+#' @examples
+#' eList <- Choptank_eList
+#' plot(eList)
+#' plot(eList, cex.main=0.7)
+plot.egret <- function(eList, ...){
```

Comment:
This is causing a warning in Travis. I think you need the first argument to be "x", since that's what `plot` has. Then, since "x" is an "eList" object, it will know to use this function.